src/user-statuses/userStatusesModel.js

```diff
@@ -21,14 +21,37 @@ import { kUserStatusZero } from './userStatusesCore';
 // Selectors.
 //
 
-const getUserStatuses = (state: PerAccountState): UserStatusesState => state.userStatuses;
+/**
+ *  The `UserStatusesState`.
+ *
+ *  Don't read from `UserStatusesState` directly; instead, pass it to
+ *  getUserStatusFromModelState.
+ */
+export const getUserStatuses = (state: PerAccountState): UserStatusesState => state.userStatuses;
 
 /**
- * The `UserStatus` object for the given UserId.
+ * The `UserStatus` object for the given UserId, from `PerAccountState`.
  */
 export const getUserStatus = (state: PerAccountState, userId: UserId): UserStatus =>
   getUserStatuses(state).get(userId, kUserStatusZero);
 
+//
+//
+// Getters.
+//
+
+/**
+ *  The `UserStatus` object for the given UserId, from `UserStatusesState`.
+ *
+ *  Use this instead of reading `UserStatusesState` directly, because it
+ *  takes care of this fact about UserStatusesState (from jsdoc):
+ *
+ *    Users who have the "zero" status, `kUserStatusZero`, may be
+ *    represented implicitly by having no record in this map.
+ */
+export const getUserStatusFromModelState = (state: UserStatusesState, userId: UserId): UserStatus =>
+  state.get(userId, kUserStatusZero);
```

Comment:
Huh interesting. In its new location there, maybe it doesn't have to have this super long name? Maybe it can just be `getUserStatus`. (It couldn't be that if it stayed in `userStatusesModel.js` because that name would collide with the existing selector there.)

Reply:
That would work well under a convention where we used these names relative to the modules they're in -- like `import * as userStatusesCore from …`. But our usual convention is instead to import the individual names directly. Under that convention, it's pretty helpful that we keep the individual names unique -- when the names are ambiguous, it gets confusing.